independent-publisher-connectors/SMS Wireless Services/apiDefinition.swagger.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
@SietseBakkerViaData thank you so much for submitting your connector with us and thank you so much for the clean code as well. 
May I ask you please to fix the indentation for all your files. all files should be properly formatted.
You can check other partners' files.
Thank you so much.